### DISCUSSION TITLE
The mitmproxy installed by pip, where is the front-end source file of mitmweb? can you help me?

### DISCUSSION THREAD

**Discussion by Pactortester** [mitmproxy/mitmproxy#5259] [Q&A] [answered]
The mitmproxy installed by pip, where is the front-end source file of mitmweb? can you help me?

I want to modify the front-end page of mitmweb locally for testing.

**Comment by mhils** [comment, marked as answer]
The precompiled assets are in https://github.com/mitmproxy/mitmproxy/tree/main/mitmproxy/tools/web/static. If you want to prototype local changes, check out https://github.com/mitmproxy/mitmproxy/tree/main/web#quick-start :)

**Reply by Pactortester** [reply]
Thank you very much, I have successfully debugged！
